.github/workflows/consistency-check.yaml

```
@@ -4,8 +4,9 @@ on:
   push:
     branches:
     - master
+    - release-1.3
   pull_request:
-    branches: [ master ]
+    branches: [ master, release-1.3 ]
```

```suggestion
    branches:
    - master
    - release-*
```

ditto for other files